content/altinn-studio/reference/ux/components/Date/_index.nb.md

```diff
@@ -0,0 +1,136 @@
+
+-->
+### Relatert
+
+- [`Number`](../number/)
+- [`Text`](../text/)
+
+## Egenskaper
+
+{{% notice warning %}}
+We are currently updating how we implement components, and the list of properties may not be entirely accurate.
+{{% /notice %}}
+
+| **Property**                 | **Type** | **Description**                                                                                                                            |
+|------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`                         | string   | Unik Id streng for komponenten                                                                                                             |
+| `value`                      | string   | Datoen du vil vise frem. Må være en ISO6801 string.                                                                                        |
+| `textResourceBindings.title` | string   | Ledeteksten til datoen du vil vise                                                                                                         |
+| `format`                     | string   | En formateringsstreng basert på [Unicode Teknisk standard](https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table)   |
+| `direction`                  | string   | Setter den strukturerte retningen for ledetekst og verdi. <br/><br/>**Enum:** [horizontal, vertical] . <br/> <br/>**Default:** horizontal. |
+| `icon`                       | string   | En URL-lenke til bildet/ikonet.                                                                                                            |
+
+## Konfigurering
+
+### Legg til komponent
+
+```json{hl_lines="6-"}
+{
+  "id": "date",
+  "type": "Date",
+  "textResourceBindings": {
+    "title": "Date"
+  },
+  "value": "2022-09-27 18:00:00.000",
+  "format": "dd.MM.yyyy",
+  "direction": "horizontal",
+  "icon": "https://www.example.com/icon.png"
+},
+```
+
+### Legg til tittel og elementer
+
+<br>
+
+#### `textResourceBindings.title`
+
+Ledetekst for datoen. Denne kan legges til som en hardkodet streng eller som en referanse til en [tekstressurs](/nb/altinn-studio/reference/ux/texts/#legge-til-og-endre-tekster-i-en-app).
+
+#### `value`
+
+Verdien av datoen som skal vises. Kan legges til som en streng eller et uttrykk.
+
+#### `format`
+
+Setter formateringen for datoen gitt i `value` feltet. Dette er basert på [Unicode Teknisk standard](https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table).
+
+#### `direction`
+
+Bestemmer om teksten vises under eller ved siden av ledeteksten. `horizontal` for ved siden av og `vertical` for under. 
+
+<!-- 
+#### `icon`
+
+Hvis du vil vise et ikon foran ledeteksten kan du legge til en url til ikonet/bildet du vil vise her.
+
+-->
+<br>
+
+#### Eksempel
+
+Dato med ledetekst og ikon.
```

Comment:
Burde hele dette eksempelet vekk?

Reply:
I så fall må det gjøres for alle tre komponenten og fjernes i property-listen i tillegg for alle tre 😊

Reply:
Endrer til å ikke inneholde icon bare